File: timBola.php
<?php

class TimBola 
{
	// deklarasi atribut
	private $namaTim = "";
	private $negaraAsal = "";
	private $tahunBerdiri = 0;
	private $namaPemain = "";
		
	// constructor
	public function __construct($namaTim = "", $negaraAsal = "", $tahunBerdiri = 0, $namaPemain = ""){
		$this -> namaTim = $namaTim;
		$this -> negaraAsal = $negaraAsal;
		$this -> tahunBerdiri = $tahunBerdiri;
		$this -> namaPemain = $namaPemain;
	}
		
	// method set & get nama tim
	public function setNamaTim($namaTim){
		$this -> namaTim = $namaTim;
	}
	public function getNamaTim(){
		return $this -> namaTim;
	}
		
	// method set & get negara asal
	public function setNegaraAsal($negaraAsal){
		$this -> negaraAsal = $negaraAsal;
	}
	public function getNegaraAsal(){
		return $this -> negaraAsal;
	}
		
	// method set & get tahun berdiri
	public function setTahunBerdiri($tahunBerdiri){
		$this -> tahunBerdiri = $tahunBerdiri;
	}
	public function getTahunBerdiri(){
		return $this -> tahunBerdiri;
	}
		
	// method set & get nama pemain
	public function setNamaPemain($namaPemain){
		$this -> namaPemain = $namaPemain;
	}
	public function getNamaPemain(){
		return $this -> namaPemain;
	}
}

?>

File: JAVA_No B/TimBola.java
public class TimBola
{
	// deklarasi atribut
	private String namaTim;
	private String negaraAsal;
	private int tahunBerdiri;
	private String namaPemain;
		
	// constructor
	public TimBola(String namaTim, String negaraAsal, int tahunBerdiri, String namaPemain){
		this.namaTim = namaTim;
		this.negaraAsal = negaraAsal;
		this.tahunBerdiri = tahunBerdiri;
		this.namaPemain = namaPemain;
	}
	public TimBola(){}
		
	// method set & get nama tim
	public void setNamaTim(String namaTim){
		this.namaTim = namaTim;
	}
	public String getNamaTim(){
		return namaTim;
	}
		
	// method set & get negara asal
	public void setNegaraAsal(String negaraAsal){
		this.negaraAsal = negaraAsal;
	}
	public String getNegaraAsal(){
		return negaraAsal;
	}
		
	// method set & get tahun berdiri
	public void setTahunBerdiri(int tahunBerdiri){
		this.tahunBerdiri = tahunBerdiri;
	}
	public int getTahunBerdiri(){
		return tahunBerdiri;
	}
		
	// method set & get nama pemain
	public void setNamaPemain(String namaPemain){
		this.namaPemain = namaPemain;
	}
	public String getNamaPemain(){
		return namaPemain;
	}
}
